configs/config.json.example

```diff
@@ -367,16 +432,10 @@
         "Any pokemon put here directly force to use Berry & Best Ball to capture, to secure the capture rate": {},
         "any": {"catch_above_cp": 1200, "catch_above_iv": 0.9, "logic": "or" },
         "Lapras": {},
-        "Moltres": {},
-        "Zapdos": {},
-        "Articuno": {},
 
         "// S-Tier pokemons (if pokemon can be evolved into tier, list the representative)": {},
-        "Mewtwo": {},
         "Dragonite": {},
         "Snorlax": {},
-        "// Mew evolves to Mewtwo": {},
-        "Mew": {},

```

Comment:
Those just appear in game events, which will me announced. Untill then, people are just sending fake reports to the broker, which triggers the sniper and waste a teleport.